metricbeat/README.md

```diff
@@ -72,22 +73,90 @@ helm repo add logzio-helm https://logzio.github.io/logzio-helm/metricbeat
 
 #### 4. Deploy
 
+You have three options for deployment:
+* [Standard configuration](#standard-config)
+* [Autodiscover configuration](#autodiscover-config)
+* [Custom configuration](#custom-config)
+
+
+<div id="standard-config">
+
+#### Deploy with standard configuration:  
 ```shell
 helm install --namespace=kube-system logzio-k8s-metrics logzio-helm/logzio-k8s-metrics
 ```
+</div>
+
+<div id="autodiscover-config">
+
+#### Deploy with Autodiscover configuration:  
+This Daemonset's default autodiscover configuration is [hints based](https://www.elastic.co/guide/en/beats/metricbeat/current/configuration-autodiscover-hints.html).
+Add your own [configuration](#autodiscover-configuration) and run:
```

Comment:
Why this line? Aren't they adding their own configuration in `custom-config`?

Reply:
It was about adding parameters. deleting this line